iot-hub/Samples/device/PnpDeviceSamples/PnpConvention/WritablePropertyResponse.cs

```diff
@@ -0,0 +1,54 @@
+﻿using Newtonsoft.Json;
```

Comment:
do we usually add the copy rights block to sample files?

Reply:
I'm not 100% sure, but it can't hurt.